tests/test_order_lines.py

```diff
@@ -19,7 +20,7 @@ def test_get_order_lines(client, response):
     assert isinstance(line, OrderLine)
     assert line.id == LINE_ID
     assert line.resource == 'orderline'
-    assert line.order_id == ORDER_ID
+    assert line.order_id == LINE_ORDER_ID
```

Comment:
Why is this changed? Why are you introducing a second ORDER_ID anyway when all tests in this file use the same fixture?